src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -199,7 +203,7 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
 
     private dumpAllCells() {
         this.notebookDocument.getCells().forEach((cell) => {
-            if (cell.kind === NotebookCellKind.Code) {
+            if (cell.kind === NotebookCellKind.Code && cell.executionSummary?.success) {
```

Comment:
this avoids the errors in dumpCell responses.

Reply:
What does that have to do with dumpCell?

Reply:
I thought only the executed cells were successful, but no, all these requests are failing. I'll remove this.